lib/Fetcher/FeedFetcher.php

```diff
@@ -146,6 +146,13 @@ public function fetch(
         );
 
         $feed->setNextUpdateTime($resource->getNextUpdate()?->getTimestamp());
+        $this->logger->debug(
+            'Feed {url} was parsed and nextUpdateTime is {nextUpdateTime}',
+            [
+            'url'   => $url,
+            'nextUpdateTime' => $resource->getNextUpdate()->getTimestamp(),
+            ]
```

Comment:
This should help with debugging the difference at least.
```suggestion
            'Feed {url} was parsed and nextUpdateTime is {nextUpdateTime} recorded as {feedUpdate}',
            [
            'url'   => $url,
            'nextUpdateTime' => $resource->getNextUpdate()->getTimestamp(),
            'feedUpdate' => $feed-> setNextUpdateTime(),
            ]
```